tools/debug/debugsymbolviewer.py

```diff
@@ -326,24 +326,35 @@ def print_wrong_sp(log_file):
 						stack_val = stack_val - int(g_stext_app[is_app_symbol - 1])
 						print_symbol(stack_addr, stack_val, is_app_symbol)
 
-# Function to Parse the i/p log file (which contains heap corruption data) and to print owner of corrupted node
+# Function to Parse the input log file (which contains heap corruption data) and to print owner of corrupted node
 def print_heap_corruption_data(log_file):
 	# Parse the contents based on tokens in log file.
 	with open(log_file) as searchfile:
 		for line in searchfile:
 			if 'allocated by code at addr' in line:
                             word = line.split(' ')
                             node = int(word[-1], 16)
-                            print_symbol_heap_corruption(node, 0)
-                            print_symbol_heap_corruption(node, 1)
+                            print_symbol_only(node, 0)
```

Comment:
we can distinguish between print_symbol and print_symbol_only.  Can we rename for easy readability